.gitignore

```diff
@@ -71,5 +71,8 @@ dist/
 
 # envsubst rendered templates
 gcpmanager-skaffold.yaml
+
+/controllermanager
+/gcpmanager
```

Comment:
won't this ignore directories like `cmd/controllermanager` and `cmd/gcpmanager` and their `internal/` counterparts?

Reply:
I don't think so because he added `/` in front of it. See relevant SO: https://stackoverflow.com/questions/24139478/when-to-use-leading-slash-in-gitignore

I didn't know about this either. The only reason I looked this up is because you asked in this PR :)

Reply:
Ahh thanks for the doublecheck there